### PR TITLE
Build/update builder

### PR DESCRIPTION
**Pull request by AxelPeter**
- Update Storybook and dependencies with minor updates
- Add `engines` field
- Update `/dist` build script

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,10 @@
   "version": "0.0.0",
   "description": "A set of maintainable components for the OVH ecosystem",
   "license": "BSD-3-Clause",
+  "engines" : {
+    "node" : ">=12.14.0",
+    "yarn": ">=1.21.1"
```
Comment (FredericEspiau) [inline package.json]: Did you try with `yarn 2`?

Comment (AxelPeter) [reply package.json]: Not really, but I could take a look

Comment (FredericEspiau) [reply package.json]: What a guy !

Comment (AxelPeter) [reply package.json]: Oh you !

**Pull request merged**